<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LlmMathQuiz</class>
 <widget class="QMainWindow" name="LlmMathQuiz">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LlmMathQuiz</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Task:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="e_task">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Answer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="e_answer">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="b_new">
        <property name="text">
         <string>New task</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="b_answer">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Answer</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
